todoman/ui.py

```diff
@@ -251,8 +259,18 @@ def detailed(self, todo):
         return rv
 
     def format_date(self, date):
+        """
+        :param datetime.date date: a datetime object
```

Comment:
In fact `format_date` recieves a datetime object (i.e. `datetime.datetime` instead of `datetime.date`). It is probably useful to add an `assert isinstance(date, datetime)` in `format_date`.